dist/t/Makefile

```diff
@@ -0,0 +1,25 @@
+.ONESHELL:
```

Comment:
This is new or was it taken from another SCM repository?

Reply:
This whole Makefile is new